help/channel-permissions.md

```diff
@@ -111,7 +112,8 @@ for details.
 
 <span class="legend_symbol">&#9726;</span><span class="legend_label">If subscribed to the channel</span>
 
-<span class="legend_symbol">&#10038;</span><span class="legend_label">
+<span class="legend_symbol">&#10038;</span>
+<span class="legend_label">
```

Comment:
From the commit message:

> This is a preparatory commit for the migration to use starlight for help center docs.

This is slightly useful, but not what's important in the commit message. What is important is explaining why this change is relevant/useful at all. I assume there's some rendering problem that happens with these spans next to each other, but somehow not with other spans a few lines before, but text explaining what the issue is with this in Starlight (so that a reviewer can reason about if there are other places that might be affected, or if the bug is somewhere else, etc.) is what we're looking for in a commit message for a preparatory change like this.

And then if you have that text, you might not need the boilerplate of the form "This is a preparatory commit for the migration", because that information would naturally be implicit in language you'd write to explain the above, like "Once we migrate to Starlight, this will fail to render because ...".

Reply:
Fixed!